src/css/application.css

```diff
@@ -118,13 +118,11 @@ body {
   flex: 0 0 auto;
   font-size: 2vh;
   font-weight: bold;
-  align-items: center;
-  padding: 0 1em;
+  align-items: stretch;
   z-index: 2;
-}
-
-.top-bar_yellow {
-  background-color: var(--color-yellow);
+  min-width: 760px;
+  color: rgba(0, 0, 0, 0.8);
+  transition: background-color .2s;
```

Comment:
So, I like this, and I am down to get rid of the yellow, but I feel like the color in the middle of the transition is kind of gross-looking. Can we use an animation to make it animate by way of an intermediate color (maybe…YELLOW…?)